python/tk_multi_publish2/dialog.py

```diff
@@ -503,7 +518,18 @@ def _create_item_details(self, tree_item):
 
         self.ui.item_description_label.setText("Description")
         self.ui.item_comments.setPlainText(item.description)
+
+        # if summary thumbnail is defined, item thumbnail should inherit it
+        # unless item thumbnail was set after summary thumbnail
+        if self._summary_thumbnail != None and not item.thumbnail_explicit:
```

Comment:
suggest removing `!= None`

Reply:
maybe instead of `thumbnail_explicit` we should rename this to `thumbnail_inherited`? then you'd have:

`if item.thumbnail_inherited and self._summary_thumbnail:`

that reads cleaner i think.